cli/popper/cli.py

```diff
@@ -95,8 +96,15 @@ def signal_handler(sig, frame):
             # Process was probably already killed, so exit silently
             pass
 
-    for cid in docker_list:
-        pu.info("Stopping container '{}'\n".format(cid))
-        pu.exec_cmd('docker stop -t 1 {}'.format(cid))
+    if len(docker_list) > 0:
+        for cid in docker_list:
+            pu.info("Stopping container '{}'\n".format(cid))
+            pu.exec_cmd('docker stop -t 1 {}'.format(cid))
+    
+    if len(singularity_list) > 0:
+        for img in set(singularity_list):
+            if os.path.isfile(img):
+                pu.info('Removing {}\n'.format(img))
+                os.remove(img)
```

Comment:
@JayjeetAtGithub this removes the container. instead, we need to stop the processes that correspond to this container. The removal of the container should be done in the `SingularityRunner.singularity_rm` function, as that is run when `--reuse` is not given.

Reply:
@ivotron In singularity, on CTRL-C, the container process is automatically stopped. It shows a message `Cleaning up.....` which is not happening now.